src/rev_ai/apiclient.py

```diff
@@ -42,11 +42,10 @@ def __init__(self, access_token):
         if not access_token:
             raise ValueError('access_token must be provided')
 
-        self.session = requests.Session()
-        self.session.headers.update({
+        self.default_headers = {
```

Comment:
The changes to this file are the important ones. We no longer have a persistent session for all requests